Formula/dislocker.rb

```diff
@@ -0,0 +1,25 @@
+class Dislocker < Formula
+  desc "FUSE driver to read/write Windows' BitLocker-ed volumes"
+  homepage "https://github.com/Aorimn/dislocker"
+  url "https://github.com/Aorimn/dislocker/archive/v0.7.1.tar.gz"
+  sha256 "742fb5c1b3ff540368ced54c29eae8b488ae5a5fcaca092947e17c2d358a6762"
+
+  bottle do
```

Comment:
Don't add `bottle do` blocks. CI will handle that as necessary.

Reply:
Removed